nautobot_floor_plan/forms.py

```diff
@@ -72,7 +62,7 @@ class FloorPlanFilterForm(NautobotFilterForm):
     field_order = ["q", "location", "x_size", "y_size"]
 
     q = forms.CharField(required=False, label="Search")
-    location = DynamicModelMultipleChoiceField(queryset=Location.objects.all(), to_field_name="slug", required=False)
+    location = DynamicModelMultipleChoiceField(queryset=Location.objects.all(), to_field_name="name", required=False)
```

Comment:
`name` is not unique. I think we need to use `pk` here.
```suggestion
    location = DynamicModelMultipleChoiceField(queryset=Location.objects.all(), to_field_name="pk", required=False)
```

Reply:
Fixed